src/twisted/plugins/twisted_reactors.py

```diff
@@ -4,59 +4,70 @@
 from __future__ import absolute_import, division
 
 from twisted.application.reactors import Reactor
-from twisted.python.compat import _PY3
+from twisted.python.reflect import requireModule
 
-default = Reactor(
+def _addReactorToAll(shortName, moduleName, description):
+    """
+
+    @param shortName: Short name of reactor
+    @param moduleName: The fully-qualified module name of the reactor
+    @param description: Description of reactor
+    @return: newly created reactor
+    @rtype: L{Reactor} or L{None}
+    """
+    newReactor = None
+    if requireModule(moduleName) is not None:
```

Comment:
@rodrigc I see.  This was confusing because you asserted that the reactors work on Windows but this PR is actually a necessary step in _making_ them work on Windows.

Why didn't you just move them into the _PY3 block?  That would be a simpler way to make this work.